app/views/users/_stats_card_balance.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="stat-card">
+  <!-- Icon -->
+  <div class="stat-card-icon">
+      <%= icon('fas', 'euro-sign') %>
+  </div>
+
+  <!-- Text -->
+  <div class="stat-card-text">
+    <div class="stat-card-title">
+      Balance
+    </div>
+
+    <%= link_to euro_from_cents(@user.balance),
+        "https://tab.zeus.gent/users/#{@user.name}",
+        target: "blank",
+        class: "stat-card-value theme-no-blend",
+        style: "color: #{get_user_balance_color(@user)}"
+    %>
+
+    <div class="stat-card-hint">
+      <%= link_to "Manage your balance",
+          "https://tab.zeus.gent/users/#{@user.name}",
```

Comment:
Dit best niet hardcoden, zodat Tab ook elders kan opgezet worden. Ook als Tab op een andere plaats wordt opgezet binnen Zeus (vroeger stond Tab op https://zeus.ugent.be/tab), dan is het op 1 plaats te veranderen